### PR TITLE
UX20-189

### PR DESCRIPTION
**Pull request by Gtonizuka**
- Added optional groupId property to submit analyses
- Fixed unit tests

### CODE REVIEW

File: src/apiServices/ClientService.ts [hunk 1]
```diff
@@ -139,8 +139,8 @@ export class ClientService {
         return await this.authService.getUsers(queryString)
     }
 
-    async getAnalysesList(): Promise<AnalysisList> {
-        return await this.analysesService.getAnalysesList()
+    async getAnalysesList(groupUUID: string): Promise<AnalysisList> {
```
Comment (nbanmp) [inline src/apiServices/ClientService.ts]: Is this parameter optional?
```suggestion
    async getAnalysesList(groupUUID?: string): Promise<AnalysisList> {
```

Comment (Gtonizuka) [reply src/apiServices/ClientService.ts]: Yes please see latest commit (I also changed naming convention)

**Comment by nbanmp** [comment]
I'm not sure if you need to update type descriptions to accept groupId.

**Comment by Gtonizuka** [comment]
After chatting to @joaosantos15 it seems that we are not gona need the GET request as the REST endpoint will change.

I reverted to old changes and will create new ticket to implement GET request once is ready

**Pull request merged**